### ISSUE TITLE
PyMISP search works correctly only with admin role

### ISSUE THREAD

**Issue by sustefil** [MISP/MISP#6318]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 20
| OS version (client)       | Ubuntu 20
| PHP version               | not sure
| MISP version / git hash   | 2.4.131
| PyMISP version | 2.4.131 

### Expected behavior
In version 2.4.128, a `read-only` user was able to use pymisp.search_index correctly (it gave the right results)
We have monitoring scripts running for months, so we now this worked prior to version 129.

### Actual behavior
Now, the search_index method works correctly ONLY with `admin`.

### Steps to reproduce the behavior

This is a call using api_key of `admin`
```
(Pdb) len(misp_2.search_index(timestamp='24h', tags=['tsec:OSINT']))
77
```

Now we changed the api_key to `read_only` user's
```
(Pdb) len(misp_2.search_index(timestamp='24h', tags=['tsec:OSINT']))
566
```

So the latter search somehow ignores the `tag` filtering and returns events with other tags.

This may be a symptom of some more serious issue

**Comment by iglocska** [comment]
I just tested it and on the latest version on HEAD it works fine. We had a bug that indeed didn't work correctly around that version, could you update to the latest and try it again?

**Comment by sustefil** [comment]
Ok, we confirm that this is fixed in 2.4.132 (tagged commit).

Thanks

Closing

**Comment by criptk** [comment]
I have a similar issue, running the update version of MISP 2.4.132. 
Some events that I have on my MISP has registry keys that are created by normal operation of spinning up internet explorer or adobe.

For example
hkey_current_user\software\microsoft\windows\currentversion\internet settings\zonemap\intranetname

If I search for this (yes its all lowercase, and as crazy as it sounds it worked for hkey_current_user\software\microsoft\windows\currentversion\internet settings\zonemap\proxybypass for the very first run, when i was runing 2.4.130. Using Admin account already.

If i search only "intranetname" it does bring me the events that has this specific registry key, then if i go into the event and copy the exact same line (eg: HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap\IntranetName) it doesn't bring me any results at all.
